### PR TITLE
Implement and refactor import component form

### PR DESCRIPTION
**Pull request by issambaccouch**

### CODE REVIEW

File: JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx [hunk 1]
```diff
@@ -17,49 +18,65 @@ import { useSnackbar } from 'notistack'
...
+      }
+    }
+  })
 
   const onDrop = (
     acceptedFiles: File[],
     fileRejections: FileRejection[]
   ): void => {
-    validate(acceptedFiles, fileRejections)
+    validate(fileRejections)
```
Comment (walisc) [inline JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx:65]: Does `enqueSnackbar` in `validate` stop the process work flow? Wont `setFileObj` still get run even though there is an error?

File: JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx [hunk 2]
```diff
@@ -69,70 +86,50 @@ const DropZone: FC = () => {
...
-        prev.status = status
-        if (status === UploadStatus.Failed) {
-          prev.progress = 0
-        }
+        setFilesObj((prev: FileObj | undefined) => {
+          if (prev?.file.name === fileObj.file.name && progressEvent.total) {
+            return {
+              ...prev,
+              progress: Math.round(
+                (progressEvent.loaded * 100) / progressEvent.total
```
Comment (walisc) [inline JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx:117]: Nice! This will be useful for users!

**Comment by walisc** [comment]
Hi @issambaccouch ...done reviewing this (only have the one comment above). Looking very cool thought Issam! Like how dynamic it is .

File: JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx (hunk 1 above)
Comment (issambaccouch) [reply JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx]: True, this issue was addressed, thank you @walisc.

**Review by walisc — APPROVED** [review]
Looks good! Thanks @issambaccouch !

**Pull request merged**